src/sql/workbench/contrib/editData/browser/media/slick.grid.css

```diff
@@ -0,0 +1,188 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.slick-header.ui-state-default, .slick-headerrow.ui-state-default, .slick-footerrow.ui-state-default  {
```

Comment:
why do we need this?  Can't we use the existing CSS for the other grids?  Also, a lot of the styles seemed to be globally scoped.